game/de.json

```diff
@@ -0,0 +1,13 @@
+{
+	"element_discovered": "Neues Element kennengelernt: {name}",
+
+	"objective": "Ziel",
+
+	"level": "Level",
+	"challenge-won-msg": "Du hast gewonnen! Du hast es entdeckt!",
```

Comment:
```suggestion
	"challenge-won-msg": "Du hast gewonnen! Du hast es entdeckt",
```

Reply:
Is it possible to have the discovered item in the middle of the sentence? That would make more sense in the german grammar

Reply:
Hm, yes, I'll modify the code to do that. Add a %item% for the item name in the right place inside the sentence, I'll modify other languages when changing the code :)

Reply:
Awesome thank you :)

Reply:
You're welcome, thank you for your translation !